client/a8c-for-agencies/sections/purchases/licenses/license-details/actions.tsx

```diff
@@ -36,6 +36,9 @@ export default function LicenseDetailsActions( {
 	const translate = useTranslate();
 
 	const [ revokeDialog, setRevokeDialog ] = useState( false );
+	// FIXME: ideally we want endpoint also sending us some slug so we could check with precission
+	const isPressableLicense = useMemo( () => product.indexOf( 'Pressable' ) !== -1, [ product ] );
```

Comment:
The `/licenses` endpoint includes the product ID associated with each license, which you can use to implement a function that tests against that field.

Reply:
Thanks! Added a tweak to `isPressableHostingProduct` and reused it.